e2e/tests/shared/shared.go

```diff
@@ -100,6 +100,57 @@ func SetupTestWithDefaults(testName string) (*Test, error) {
 		Provisioner:        config.Provisioner,
 	}
 
+	if utiliyManaged == "argocd" {
+		createClusterReq = &model.CreateClusterRequest{
+			AllowInstallations: true,
+			Annotations:        testAnnotations(testID),
+			AMI:                config.KopsAMI,
+			VPC:                config.VPC,
+			Provisioner:        config.Provisioner,
```

Comment:
Instead of recreating the request we can just alter the existing one to change or add the values we want.